### ISSUE TITLE
Document minimum CSS needed to style arrows

### ISSUE THREAD

**Issue by brianmhunt** [FezVrasta/popper.js#674]
### Steps to reproduce the problem

1.  Install Tooltip.js
2.  Look for docs on how to add styled arrows

### What is the expected behavior?

It would help to have some CSS docs linked from either Tooltip.js, or the README, or somewhere, indicating what minimum styling is needed for tooltips with styled arrows.

### What went wrong?

Documentation of this sort does not appear to be available anywhere.

### Any other comments?

:beers:

**Comment by kgrote** [comment]
Agree. Took me a while to realize you have to manually apply `position: absolute` for the arrow positioning to work correctly.

**Comment by seyfer** [comment]
Struggling to add arrow border...

**Comment by bfeifke** [comment]
> Agree. Took me a while to realize you have to manually apply `position: absolute` as well as top/left/right/bottom alignment for the arrow positioning to work correctly.

@kgrote can you elaborate on what you mean?

**Comment by robclancy** [comment]
The fact this isn't out of the box was incredible when installing this. I had to grab the styles from the demo site. Such a weird decision... weird enough to even think of it let alone do it.

**Comment by kinesias** [comment]
The installing process of popper.js is very painful. No css included, very bad documentation with almost no examples. It took me 3 hours to get this plugin running.

**Comment by FezVrasta** [comment]
I find it awesome this issue has stayed open for 9 months and nobody sent a PR to add this piece of documentation.

**Comment by robclancy** [comment]
I find it awesome that this issue stayed open for 9 months and you only just comment now. And when you do you play victim.

**Comment by FezVrasta** [comment]
I added labels to this issue 9 months ago, I don't need to comment, the labels explain everything 👌

**Comment by FezVrasta** [comment]
Fixed in 8f2469111dcc483623a5fcba99e34c661f9ea74d

**Comment by aalimovs** [comment]
Anyone looking to have a triangle arrow like in the examples on the site, using the example above, also add:

```
#pop [x-arrow]::before {
position: absolute;
    width: 10px;
    height: 10px;
    background-color: red;
    content: '';
    -webkit-transform: rotate(45deg);
    -ms-transform: rotate(45deg);
    transform: rotate(45deg);
}
```

And update the negative -10px margins to -5px. Also remove the `background-color` on `#pop [x-arrow]`.
